workflow-manager/main.go

```diff
@@ -261,8 +261,24 @@ func main() {
 
 	log.Printf("found %d peer validations", len(peerValidationBatches))
 
-	// TODO: Check that both peer and own have something.
-	var aggregationBatches = peerValidationBatches
+	// We can only aggregate a batch if we have our own validations and the peer's validations. To
+	// that end we construct a set of own validations, and check each peer validation against it. Go
```

Comment:
nit: maybe make a mention that we're making an intersection of ownValidations and peerValidations to get the aggregationBatches as a tl;dr